examples/local/fivenodenetwork/main.go

```diff
@@ -66,7 +67,25 @@ func main() {
 
 func run(log logging.Logger, binaryPath string) error {
 	// Create the network
-	nw, err := local.NewDefaultNetwork(log, binaryPath)
+	config := local.NewDefaultConfig(binaryPath)
+	for i := range config.NodeConfigs {
+		config.NodeConfigs[i].ConfigFile = []byte(`{
+	"network-peer-list-gossip-frequency":"250ms",
+	"network-max-reconnect-delay":"1s",
+	"public-ip":"127.0.0.1",
+	"health-check-frequency":"2s",
+	"api-admin-enabled":true,
+	"api-ipcs-enabled":true,
+	"index-enabled":true
+}`)
+		wr := &writer{colIdx: i, name: fmt.Sprintf("node%02d", i+1), f: os.Stdout}
+		config.NodeConfigs[i].ImplSpecificConfig = local.NodeConfig{
+			BinaryPath: binaryPath,
+			Stdout:     wr,
+			Stderr:     wr,
```

Comment:
If we provide `Stdout` and `Stderr`, it'll print all the logs from each node to the terminal, which is kind of messy. I'm OK with this change if we change the log display level to `ERROR` in each node config (and keep the log level written to disk at the default, `INFO`.)